includes/compat/trait-wc-stripe-subscriptions.php

```
@@ -1267,22 +1267,33 @@ public function update_subscription_payment_method_from_order( $order, $payment_
 	 *
 	 * @param bool     $editable The current editability of the subscription.
 	 * @param WC_Order $order    The order object.
 	 * @return bool Returns true if the subscription can be edited, false otherwise.
 	 */
 	public function disable_subscription_edit_for_india( $editable, $order ) {
+		if ( ! WC_Stripe_Subscriptions_Helper::is_subscriptions_enabled() || ! $this->is_subscription( $order ) ) {
+			return $editable;
+		}
+
+		// If the order doesn't have a parent order, we can't check the payment method, so allow editing.
 		$parent_order = wc_get_order( $order->get_parent_id() );
```

PHPMD (2.15.0)

[error] The variable $parent_order is not named in camelCase. (undefined)

(CamelCaseVariableName)